wire/internal/wire/wire_test.go

```diff
@@ -41,7 +42,7 @@ func TestWire(t *testing.T) {
 	}
 	// The marker function package source is needed to have the test cases
 	// type check. loadTestCase places this file at the well-known import path.
-	wireGo, err := ioutil.ReadFile(filepath.Join("..", "..", "wire.go"))
+	wireGo, err := readFile(filepath.Join("..", "..", "wire.go"))
```

Comment:
Not sure if this one needs to be converted....?

Reply:
If the Go source file can't be loaded as-is by the build, then that indicates a legitimate failure. As such, I think the correct behavior to be to preserve the bytes.